psalm.xml

```diff
@@ -17,6 +17,13 @@
         </ignoreFiles>
     </projectFiles>
     <issueHandlers>
+        <DeprecatedConstant>
+            <errorLevel type="suppress">
+                <file name="lib/Doctrine/ORM/Configuration.php"/>
+                <file name="lib/Doctrine/ORM/Query/Lexer.php"/>
+                <file name="lib/Doctrine/ORM/Query/Parser.php"/>
+            </errorLevel>
+        </DeprecatedConstant>
```

Comment:
This block should be moved under the `<DeprecatedClass>` block to maintain the alphabetical order.